src/comm_manager.cpp

```diff
@@ -397,37 +397,30 @@ void CommManager::send_rc_raw(void)
 
 void CommManager::send_diff_pressure(void)
 {
-  if (RF_.sensors_.data().diff_pressure_valid) {
     comm_link_.send_diff_pressure(sysid_, RF_.sensors_.data().diff_pressure_velocity,
                                   RF_.sensors_.data().diff_pressure,
                                   RF_.sensors_.data().diff_pressure_temp);
   }
-}
 
 void CommManager::send_baro(void)
 {
-  if (RF_.sensors_.data().baro_valid) {
     comm_link_.send_baro(sysid_, RF_.sensors_.data().baro_altitude,
                          RF_.sensors_.data().baro_pressure, RF_.sensors_.data().baro_temperature);
   }
-}
 
 void CommManager::send_sonar(void)
 {
-  if (RF_.sensors_.data().sonar_range_valid) {
     comm_link_.send_sonar(sysid_,
                           0, // TODO set sensor type (sonar/lidar), use enum
                           RF_.sensors_.data().sonar_range, 8.0, 0.25);
   }
-}
 
 void CommManager::send_mag(void)
 {
-  if (RF_.sensors_.data().mag_present) comm_link_.send_mag(sysid_, RF_.sensors_.data().mag);
+ 	  	  comm_link_.send_mag(sysid_, RF_.sensors_.data().mag);
```

Comment:
This has tabs instead of spaces. The ROS style guide directs to use spaces instead of tabs, and mixing them makes for very messy formatting. clang-format can fix this for you if you'd like.

Reply:
ran clang-tools on that file.